### PR TITLE
#174adding images

### PR DESCRIPTION
**Pull request by BelousSofiya**
Uploading banner_image. User can add image to his company and get image of every company. Only BE part. 
image
image

### CODE REVIEW

File: images/views.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    RetrieveUpdateAPIView,
+)
+from rest_framework.parsers import MultiPartParser, FormParser
+
+from profiles.permissions import UserIsProfileOwnerOrReadOnly
+from profiles.models import Profile
+from .serializers import BannerSerializer
+
+
+class BannerChangeAPIView(RetrieveUpdateAPIView):
```
Comment (Shushunya) [inline images/views.py:11]: I think it's better to rename the view. Something like `BannerRetrieveUpdate` sounds good for me.

**Review by Shushunya — APPROVED** [review]

File: images/views.py (hunk 1 above)
Comment (BelousSofiya) [reply images/views.py]: Thank you for feedback. Done.

File: images/tests/test_banner.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        self.wrong_image = self._generate_image("png", (7000, 7000))
+
+        self.user = UserFactory(email="test1@test.com")
+        self.company_dnipro = ProfileStartupFactory.create(
+            person=self.user,
+            name="Dnipro",
+            banner_image=f"banners/{self.right_image.name}",
+        )
+
+        self.company_kiev = ProfileCompanyFactory(name="Kyivbud")
```
Comment (ProDG) [inline images/tests/test_banner.py:40]: #KyivNotKiev. :)

**Review by ProDG — APPROVED** [review]

File: images/tests/test_banner.py (hunk 2 above)
Comment (BelousSofiya) [reply images/tests/test_banner.py]: Thank you for feedback. Done.

**Pull request merged**